src/NServiceBus.RavenDB/SagaPersister/RavenDbSagaStorage.cs

```diff
@@ -2,22 +2,13 @@
 {
     using NServiceBus.SagaPersisters.RavenDB;
 
-    /// <summary>
-    ///     RavenDB Saga Storage.
-    /// </summary>
-    public class RavenDbSagaStorage : Feature
+    class RavenDbSagaStorage : Feature
```

Comment:
Let's obsolete this one here, and move it to a different namespace.

Wonder if we should move all the features to NServiceBus.RavenDB or something for consistency anyway? This seems like a very odd namespace to use, not being inside NServiceBus.RavenDB.